CMakeLists.txt

```diff
@@ -55,6 +55,8 @@ PROJECT(${PROJECT_NAME} ${PROJECT_ARGS})
 # If needed, fix CMake policy for APPLE systems
 APPLY_DEFAULT_APPLE_CONFIGURATION()
 
+CHECK_MINIMAL_CXX_STANDARD(14 ENFORCE)
```

Comment:
If I set my environment to 17, will this macro cause a conflict or will it gracefully accept it since 17>14?

Reply:
Based on @nim65s's logic here: https://github.com/jrl-umi3218/jrl-cmakemodules/blob/master/cxx-standard.cmake#L64, I would say that 14< should be allowed.